### PR TITLE
Document the file() controller helper

### PR DESCRIPTION
**Pull request by javiereguiluz**
This rewords #6732 to document the new feature with less words and more code.

### CODE REVIEW

File: book/controller.rst [hunk 1]
```diff
@@ -806,11 +812,38 @@ If the :doc:`serializer service </cookbook/serializer>` is enabled in your
...
+    {
+        // load the file from the filesystem
+        $file = new File('/path/to/some_file.pdf');
+        return $this->file($file);
+
+        // rename the downloaded file
+        return $this->file($file, 'custom_name.pdf');
+
+        // display the file contents in the browser instead of downloading it
+        $this->file('invoice_3241.pdf', 'my_invoice.pdf', ResponseHeaderBag::DISPOSITION_INLINE);
```
Comment (WouterJ) [inline book/controller.rst]: missing `return`

Comment (javiereguiluz) [inline book/controller.rst]: Fixed. Thanks.

**Comment by xabbuh** [comment]
👍 

Status: Reviewed

**Comment by WouterJ** [comment]
:+1: Thanks!

**Pull request closed without merging**